homeworks/george-borschevskiy/homework-1/index-2.js

```diff
@@ -0,0 +1,4 @@
+const firstValue = prompt('input first value')
+const secondValue = prompt('input second value')
+
+firstValue == secondValue ? alert(true) : alert(false)
```

Comment:
Better to use if else here, ask me why on lesson 

```suggestion
if (firstValue == secondValue) {
  alert(true)
} else {
  alert(false)
}
```